Sources/Rendering/OpenGL/RenderWindow/index.d.ts

```diff
@@ -270,7 +270,7 @@ export interface vtkOpenGLRenderWindow extends vtkOpenGLRenderWindowBase {
 	 * @param {Number} numComps 
 	 * @param {Boolean} useFloat 
 	 */
-	getDefaultTextureInternalFormat(vtktype: VtkDataTypes, numComps: number, useFloat: boolean): void;
+	getDefaultTextureInternalFormat(vtktype: VtkDataTypes, numComps: number, oglNorm16Ext?:unknown): void;
```

Comment:
Minor nit: space `:unknown` to `: unknown` (space). We don't have the linter turned on yet for d.ts files (and we should).

Reply:
done